src/assets/apple-ios.svg

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I think that for versioning purposes, it's easier just to have them embedded with the widget. Otherwise, we'd have to start versioning the external assets.